mobile/src/main/java/rs/readahead/washington/mobile/views/activity/SettingsActivity.java

```diff
@@ -44,10 +48,6 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.toolbar.setStartTextTitle(getResources().getString(R.string.settings_app_bar));
         setSupportActionBar(binding.toolbar);
 
-        binding.toolbar.setBackClickListener(() -> {
```

Comment:
Why was this deleted?

Reply:
thank you djordje, this should not be deleted, I will add it